Fixed problem with BulkEditingTools and Tabs in IE

pschiffe opened the pull request:
When GridFieldBulkEditingTools are used on GridField within Form Tab, Internet
Explorer (11) has problems switching to another Form Tab. In this case, clicking
on different tab has no effect, one must at first click somewhere else on the
page, and only after that it's possible to change the tab. While debugging, I
found out that the reason of this behavior is cloning, moving and removing of
the tr with the bulk manager options. Without cloning and removing, and moving
the tr with filter header or sortable header instead of tr with bulk manager
options, switching tabs works fine even in the IE 11.

colymba commented:
Does this affect any other browser? Do you know where the issue came from? `this.clone()` maybe?

pschiffe commented:
I didn't see it in any other browser, but I'm using only Firefox, Chrome and IE. The issue actually came from `$component.insertBefore($tr.eq(newIndex));` and it was there even when I removed cloning (and replaced `$component` with `this`). That's why I did it other way around with `$tr.eq(newIndex).insertAfter($(this));`
